module/Database/view/database/member/subscribe.phtml

```diff
@@ -428,6 +429,13 @@ schrijven overeenkomstig de opdracht van GEWIS.') ?>
 
 <?= $this->form()->closeTag() ?>
 </div>
+
+<script>
+    window.addEventListener('load', (event) => {
+        validator = new InputValidator(document.getElementById("<?= $form->getAttribute('id'); ?>"));
+    });
+</script>
+
```

Comment:
While I do like the idea of having some things automatically be updated on the front-end it does create an additional point of maintenance when we are updating the enrolment form.

If we really want to keep it we should either ask the user if they want to change update the value instead of doing it automatically (see comment on the JS file.)

Reply:
[3f30a22](https://github.com/GEWIS/gewisdb/pull/186/commits/3f30a22c6a68489ac3ecb2665e4ea7d950ae38f2)